Controls/Mail/Mailman.c

```diff
@@ -21,7 +36,16 @@ void depositLetter(void) {
  */
 void Task_Mailman(void* p_arg) {
 	OS_ERR err;	// Make sure to check for errors and print the error code if not OS_ERR_NONE
+	//CPU_TS ts; 
+	
```

Comment:
Something that might be nice is to seed your random number generation with srand and the current time. This makes it so that your random number generator will not return the same numbers on every run of the program. https://www.geeksforgeeks.org/rand-and-srand-in-ccpp/